config/sql/config.go

```diff
@@ -60,6 +61,8 @@ func msSQLConnectionDetails(attr map[string]interface{}) (map[string][]byte, err
 func Configure(p *config.Provider) {
 	p.AddResourceConfigurator("azurerm_sql_server", func(r *config.Resource) {
 		r.Version = common.VersionV1Alpha2
+		r.TerraformResource.Schema["extended_auditing_policy"].Elem.(*schema.Resource).
```

Comment:
Do we know under what exact or close conditions we see this information lost? I'd love us to have some guidance about this and let people know under what conditions this might happen.

Reply:
As @turkenh mentioned, we only observed this with nested blocks. I'm not sure if there is a pattern or not.